setup.py

```diff
@@ -12,5 +15,6 @@
       author='Avants, Gosselin, Tustison, Reardon',
       author_email='stnava@gmail.com',
       license='Apache 2.0',
+      install_requires=requirements,
```

Comment:
The requirements file is converted into a list and fed to the setup function. Again, some might call this blasphemous but I believe your intention for the reqs file is the base dependencies for this package to operate, which aligns with the same goals as install_requires. 

Now when someone `pip install antspymm`, the packages in the the requirements file will be added to the dependency graph.